test/e2e/ingress.go

```diff
@@ -274,13 +278,19 @@ func (cont *IngressController) create() {
 		Expect(err).NotTo(HaveOccurred())
 	}
 	rc := rcFromManifest(cont.rcPath)
-	existingRc, err := cont.c.ReplicationControllers(api.NamespaceSystem).Get(lbContainerName)
+
+	listOpts := api.ListOptions{LabelSelector: labels.SelectorFromSet(labels.Set(clusterAddonLBLabels))}
+	existingRCs, err := cont.c.ReplicationControllers(api.NamespaceSystem).List(listOpts)
 	Expect(err).NotTo(HaveOccurred())
+	if len(existingRCs.Items) == 0 {
```

Comment:
If I did find multiple in the kube-system ns, taking the first one is ok I think, give that we're just using it to get the spec. However finding multiple would be weird, so modified.